src/s1reader/s1_reader.py

```diff
@@ -349,28 +394,47 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
 
-    #parse manifest.safe to retrieve IPF version
+    # parse manifest.safe to retrieve IPF version
     manifest_path = os.path.dirname(annotation_path).replace('annotation','') + 'manifest.safe'
     with open_method(manifest_path, 'r') as f_manifest:
         tree_manfest = ET.parse(f_manifest)
         ipf_version = get_ipf_version(tree_manfest)
 
-    #Load the Product annotation - for EAP calibration
-    #with open_method(annotation_path, 'r') as f_lads:
-    #    tree_lads = ET.parse(f_lads)
-    #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
+    # Load the Product annotation - for EAP calibration
+    with open_method(annotation_path, 'r') as f_lads:
+        tree_lads = ET.parse(f_lads)
+        product_annotation = ProductAnnotation.from_et(tree_lads)
 
-    #load the Calibraton annotation
-    calibration_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
+    # load the Calibraton annotation
+    calibration_annotation_path =\
+        annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
     with open_method(calibration_annotation_path, 'r') as f_cads:
         tree_cads = ET.parse(f_cads)
-        calibration_annotation = s1_annotation.CalibrationAnnotation.from_et(tree_cads)
+        calibration_annotation =\
+            CalibrationAnnotation.from_et(tree_cads,
+                                          calibration_annotation_path)
 
-    #load the Noise annotation
+    # load the Noise annotation
     noise_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/noise-')
     with open_method(noise_annotation_path, 'r') as f_nads:
         tree_nads = ET.parse(f_nads)
-        noise_annotation = s1_annotation.NoiseAnnotation.from_et(tree_nads, ipf_version)
+        noise_annotation = NoiseAnnotation.from_et(tree_nads, ipf_version,
+                                                   noise_annotation_path)
+
+    # load AUX_CAL annotation
+    flag_apply_eap = is_eap_correction_necessary(ipf_version)
+    if flag_apply_eap.phase_correction:
+        path_aux_cals = f'{os.path.dirname(s1_annotation.__file__)}/data/aux_cal'
```

Comment:
After installation using conda, will `data/aux_cal` be installed as a sub-directory of `os.path.dirname(s1_annotation.__file__)`? Notice that conda may change the tree structure of your repo, separating executables, modules, and data.
Also, please use `os.path.join()` for directory/file separator rather than forward slashes.

Reply:
`path_aux_cals` is indeed a path in the file system. Therefore I agree that it makes more sense to use `os.path.join()` here. I've modified & tested this line.